commons/libs/constants/Environments.ts

```diff
@@ -1,3 +1,3 @@
-type Environments = 'local' | 'dev' | 'qa' | 'production' | 'test'
+enum Environments { LOCAL = 'local', DEV = 'dev', QA = 'qa', PRODUCTION = 'production', TEST = 'test' }
```

Comment:
Well. Maybe In future we need to pass this from terraform variable.

Reply:
Got it vai.